skins/oasis/css/core/ads.scss

```diff
@@ -19,6 +19,7 @@ $uap-progress-bar-color: #b7b7b7;
 $uap-progress-bar-current-time-color: #666;
 $uap-video-animation-duration: 0.4s;
 $uap-video-player-button-size: 24px;
+$wikia-bar-height: 25px;
```

Comment:
What do you think of declaring this variable in [WikiaBar.scss](https://github.com/Wikia/app/blob/b03df0a89ed672697e9c130d529bf1eb25f49cda/extensions/wikia/WikiaBar/css/WikiaBar.scss#L27) and re-using it here?